engine-tests/src/main/java/org/terasology/engine/core/PathManagerProvider.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.core;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.api.extension.ParameterResolver;
+
+import static org.mockito.Mockito.mock;
+
+public class PathManagerProvider implements ParameterResolver {
```

Comment:
TIL about parameter resolvers being a thing. 

I'd highly appreciate docstrings for this class to explain what it does and how to use it.